osu.Game/Skinning/SkinConfiguration.cs

```diff
@@ -7,28 +7,25 @@
 
 namespace osu.Game.Skinning
 {
+    /// <summary>
+    /// An empty skin configuration.
+    /// </summary>
     public class SkinConfiguration : IHasComboColours, IHasCustomColours
     {
         public readonly SkinInfo SkinInfo = new SkinInfo();
 
-        public List<Color4> ComboColours { get; set; } = new List<Color4>
-        {
-            new Color4(17, 136, 170, 255),
-            new Color4(102, 136, 0, 255),
-            new Color4(204, 102, 0, 255),
-            new Color4(121, 9, 13, 255)
-        };
+        public List<Color4> ComboColours { get; set; } = new List<Color4>();
 
         public Dictionary<string, Color4> CustomColours { get; set; } = new Dictionary<string, Color4>();
 
-        public string HitCircleFont { get; set; } = "default";
+        public string HitCircleFont { get; set; }
 
         public int HitCircleOverlap { get; set; }
 
         public float? SliderBorderSize { get; set; }
 
         public float? SliderPathRadius { get; set; }
 
-        public bool? CursorExpand { get; set; } = true;
```

Comment:
Doesn't this now mean that the default for an `OsuLegacySkin` is now `false`? It doesn't seem to have any form of fallback if null...